### ISSUE TITLE
[bug] Failed to assemble APK

### ISSUE THREAD

**Issue by johannesschaffer** [tauri-apps/tauri#12029]
### Describe the bug

When I'm running `npm run tauri android dev` it fails with: `Failed to assemble APK: command ["C:\\Users\\johan\\Programming\\tests\\chats-tauri\\src-tauri\\gen/android\\gradlew.bat", "--project-dir", "C:\\Users\\johan\\Programming\\tests\\chats-tauri\\src-tauri\\gen/android"] exited with code 1`

### Reproduction

I'm just using the vite + typescript template

### Full `tauri info` output

```text
[✔] Environment
    - OS: Windows 10.0.26100 x86_64 (X64)
    ✔ WebView2: 131.0.2903.112
    ✔ MSVC: Visual Studio Build Tools 2022
    ✔ rustc: 1.83.0 (90b35a623 2024-11-26)
    ✔ cargo: 1.83.0 (5ffbef321 2024-10-29)
    ✔ rustup: 1.27.1 (54dd3d00f 2024-04-24)
    ✔ Rust toolchain: stable-x86_64-pc-windows-msvc (default)
    - node: 22.11.0
    - npm: 10.9.0

[-] Packages
    - tauri 🦀: 2.1.1
    - tauri-build 🦀: 2.0.3
    - wry 🦀: 0.47.2
    - tao 🦀: 0.30.8
    - @tauri-apps/api : 2.1.1
    - @tauri-apps/cli : 2.1.0

[-] Plugins
    - tauri-plugin-notification 🦀: 2.2.0
    - @tauri-apps/plugin-notification : 2.2.0

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:1420/
    - framework: React
    - bundler: Vite
```

**Comment by johannesschaffer** [comment]
Same problem on macOS. Installing Java 17 solved it

**Comment by FabianLars** [comment]
Closing as solved then? If it's not fixed on Windows yet, interpret this as closed as a duplicate of #11502
